src/mobile-ui-react/PanelHeader.tsx

```diff
@@ -41,6 +41,11 @@ export interface PanelHeaderButtonProps extends CommonProps {
   onClick: (e: React.MouseEvent) => void;
 }
 
+function IsPanelHeaderButtonProps(obj?: any): obj is PanelHeaderButtonProps {
```

Comment:
Why is this capitalized? (I would expect eslint to complain about that; what am I missing?)

Reply:
I fixed the capitalization. I don't know why there were no eslint errors, maybe we don't have something configured as expected? I didn't see any problems in VSCode nor when running `npm run lint`.